src/main/kotlin/no/nav/helse/sporenstreks/integrasjon/rest/aareg/MockAaregArbeidsforholdClient.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.helse.sporenstreks.integrasjon.rest
+
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.AaregArbeidsforholdClient
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.Ansettelsesperiode
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.Arbeidsforhold
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.Arbeidsgiver
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.Opplysningspliktig
+import no.nav.helse.arbeidsgiver.integrasjoner.aareg.Periode
+import no.nav.helse.sporenstreks.domene.Arbeidsgiverperiode
+
+class MockAaregArbeidsforholdClient : AaregArbeidsforholdClient {
+    override suspend fun hentArbeidsforhold(ident: String, callId: String): List<Arbeidsforhold> =
+        listOf(
+            Arbeidsforhold(
+                Arbeidsgiver("test", "810007842"),
```

Comment:
Har du andre orgnummer du kan bruke? Som helt åpenbart er fake...

Reply:
organisasjonsnummeret blir validert av `OrganisasjonsnummerValidator`, så jeg kan ikke sette inn hva som helst.

Reply:
det er mulig å lage tester for individuelle valideringsmetoder som [her](https://github.com/navikt/fritakagp/blob/5121fee8521a08539a05ced78cabae3857e19598/src/test/kotlin/no/nav/helse/fritakagp/web/api/resreq/validation/AaregConstraintsKtTest.kt#L174-L191):